test/project_types/extension/tasks/execute_commands/build_test.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+require "test_helper"
+
+module Extension
+  module Tasks
+    module ExecuteCommands
+      class BuildTest < MiniTest::Test
+        def setup
+          super
+          ShopifyCLI::ProjectType.load_type(:extension)
+        end
+
+        def test_error_is_raised_if_error_occurs
```

Comment:
Reworked these tests, since this class is essentially just gathering data and forwarding it along, I will just be testing we are returning a Result object.